pkgs/applications/kde/kdeconnect-kde.nix

```diff
@@ -55,6 +58,7 @@ mkDerivation {
     qtquickcontrols2
     qtx11extras
     wayland
+    wayland-protocols
```

Comment:
one of the wayland-* packages goes to nativeBuildInputs

Reply:
That's wayland-scanner but this is probably out of scope for this PR, as cross is very likely broken all over for KDE stuff.